### ISSUE TITLE
Add All GCD Project to Map

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/gcd#157]
Did we consciously decide (because of how big projects can now get with batching) NOT to have `Add GCD Project to the Map`? My preference is that it is there. If it scares you, you could always put a warning dialog for when over 20 layers exist that reads `Warning, you have over 20 layers in the GCD Project. Adding all to the Map could be slow. Are you sure you want to proceed?`. Thoughts?

**Comment by philipbaileynar** [comment]
@joewheaton yeah I like your suggestion. This is not too hard to reimplement.
